packages/components/src/styles/common-styles.ts

```diff
@@ -147,7 +179,13 @@ export const getFrostedGlassBackgroundJssStyles = (
             backdropFilter: 'blur(0px)',
             WebkitBackdropFilter: 'blur(0px)',
           }),
-      transition: `opacity ${duration} ${timingFn}, backdrop-filter ${duration} ${timingFn}, --webkit-backdrop-filter ${duration} ${timingFn}`,
+      transition: `${getTransition('opacity', duration, timingFn)}, ${getTransition(
+        'backdrop-filter',
+        duration,
+        timingFn
+      )}, ${getTransition('opacity', duration, timingFn)}, --webkit-backdrop-filter ${motionDurationMap[duration]} ${
```

Comment:
Why not use the `getTransition` utility for `--webkit-backdrop-filter` as the last one?